Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -101,25 +112,25 @@ class PostRideSummaryViewController: UIViewController {
         
         let startingLocationSection = ImageLabelView()
         startingLocationSection.label.font = .systemFont(ofSize: 18)
-        startingLocationSection.label.text = "Ithaca, NY"
+        startingLocationSection.label.text = ride.departureLocation
         startingLocationSection.imageView.image = UIImage(systemName: "paperplane", withConfiguration: UIImage.SymbolConfiguration(pointSize: 36))
```

Comment:
factor out the images into a file for static declarations of all the UIImage's or something similar. this would be in a future PR though

you can reference the way Volume did it [here](https://github.com/cuappdev/volume-ios/blob/master/Volume/Extensions/Image%2BExtensions.swift)